AirCasting/CoreData/MeasurementStreamStorage.swift

```diff
@@ -62,7 +64,14 @@ final class CoreDataMeasurementStreamStorage: MeasurementStreamStorage {
         updateSessionParamsService.updateSessionsParams(sessionEntity, session: session)
         return try createMeasurementStream(for: sessionEntity, context: context, stream)
     }
-
+    
+    func existingMeasurementStream(_ sessionUUID: SessionUUID, name: String) throws -> MeasurementStreamLocalID? {
+        let context = persistenceController.editContext()
+        let session = try? context.existingSession(uuid: sessionUUID)
```

Comment:
I think we want `try` here instead of `try?` b/c it will ignore the error. If we want to ignore the error this functions `throws` is unneeded